ISM43362Interface.cpp

```diff
@@ -51,23 +46,21 @@ ISM43362Interface::ISM43362Interface(bool debug)
     thread_read_socket.start(callback(this, &ISM43362Interface::socket_check_read));
 
     _mutex.lock();
-    const char *read_version;
 
     _ism.setTimeout(ISM43362_MISC_TIMEOUT);
 
     // Check all supported firmware versions
-    read_version = _ism.get_firmware_version();
+    _FwVersion = _ism.get_firmware_version();
 
-    if (!read_version) {
+    if (!_FwVersion) {
         error("ISM43362Interface: ERROR cannot read firmware version\r\n");
     }
-    debug_if(_ism_debug, "ISM43362Interface: read_version = [%s]\r\n", read_version);
-
-    if ((strcmp(read_version, supported_fw_versions[0]) == 0) || (strcmp(read_version, supported_fw_versions[1]) == 0)) {
-        debug_if(_ism_debug, "ISM43362Interface: firmware version is OK\r\n");
-    } else {
-        debug_if(_ism_debug, "ISM43362Interface: WARNING this firmware version has not been tested !\r\n");
+    debug_if(_ism_debug, "ISM43362Interface: read_version = %lu\r\n", _FwVersion);
+#if TARGET_DISCO_L475VG_IOT01A
+    if (_FwVersion < LATEST_FW_VERSION_NUMBER) {
+        debug_if(_ism_debug, "ISM43362Interface: please update FW\r\n");
     }
```

Comment:
Update firmware may not be required. But at least inform user that there is more recent one.
"ISM43362Interface: consider upgrading to latest FW " then display version number